src/simulator/halo_simulator.py

```diff
@@ -228,12 +229,40 @@ def simulated_venn_diagram_reach_by_spend(
               satisfying the request.
             max_frequency:  The maximum frequency for which to report reach.
         Returns:
-            A list of ReachPoint. Each reach point represents the mapping from 
+            A list of ReachPoint. Each reach point represents the mapping from
             the spends of a subset of publishers to the differentially private
             estimate of the number of people reached in this subset.
         """
         raise NotImplementedError()
 
+    def _aggregate_reach_in_venn_diagram_regions(
+        self, pub_ids: List[int], venn_diagram_regions: Dict[int, List]
+    ) -> int:
+        """Returns an aggregated reach from Venn diagram regions given publishers.
+
+        With the Venn diagram regions, we aggregate reach from the regions which
+        contain any one of given publishers. Note that the binary representation of
```

Comment:
Would you consider rephrasing this sentence. An ideas is as follows.

To obtain the union reach of the given subset of publishers, we sum up the reach from the regions of which the binary representation equals one at the digit corresponding to any of the given publishers.

Reply:
Move the discussion to the other [conversation](https://github.com/world-federation-of-advertisers/planning-evaluation-framework/pull/49#discussion_r670725798) to avoid separate discussion.